lib/apps.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const ApiGroup = require('./api-group');
+
+class Apps extends ApiGroup {
+  constructor(options) {
+    const genericTypes = [
+      'petsets'
+    ];
+    options = Object.assign({}, options, {
+      path: 'apis/apps',
+      version: options.version || 'v1beta1',
```

Comment:
How should we handle the petset -> statefulset rename [1]?

The simplest thing would be to add `statefulset` to the `genericTypes` array and add a comment about the rename.  Any better ideas?

[1]: https://github.com/kubernetes/kubernetes/pull/35663